src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/MessageList.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-list lines="two" class="flex-grow-1 message-list">
+  <v-list lines="two" class="flex-grow-1 message-list no-padding">
```

Comment:
similar comment to the above for to avoid putting it into global styles. Unless we are using this in several areas.

Reply:
Agreed, Done.